bindings/python/setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import find_packages, setup, Extension
+from Cython.Build import cythonize
+
+import os
+
+
+LIBBSON_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), 'pymongoarrow/libbson/lib')
```

Comment:
Does this have to be an absolute path? If we make this a relative path (and include the libbson file in the wheel) will cython work correctly?

Reply:
It seems this will not be sufficient. The correct way to distribute shared library objects is using delocate or audit-wheel as I have mentioned in my comment.